lib/expand/phases/wait.go

```diff
@@ -60,7 +61,10 @@ type waitPlanetExecutor struct {
 func (p *waitPlanetExecutor) Execute(ctx context.Context) error {
 	p.Progress.NextStep("Waiting for the planet to start")
 	p.Info("Waiting for the planet to start.")
-	err := utils.Retry(defaults.RetryInterval, defaults.RetryAttempts,
+	ctx, cancel := defaults.WithTimeout(ctx)
+	defer cancel()
+	b := backoff.NewConstantBackOff(defaults.RetryInterval)
+	err := utils.RetryWithInterval(ctx, b,
```

Comment:
This fixes the issue with being unable to interrupt this phase otherwise (i.e. cancelling the context did not have an effect).